### ISSUE TITLE
Screen reader support

### ISSUE THREAD

**Issue by ksb749** [jellyfin/jellyfin-vue#2474]
### Description of the bug

There is somethings that work really great, but in general there is missing some alternative texts like when playing media, there is many buttons that are missing alternative text.
Is possible, it would be nice with keyboard shortcuts to navigate, when tapping around with Orca on Debian 12.7 in a browser, it takes very long time and a lot of tapping. And when playing media, the buttons does disappear when tapping around.

I havent tested on Windows, android or Iphones, just in Firefox on Debian 12.7 with Firefox

### Steps to reproduce

Open a browser, go to your jellyfin-vue adress and enable Orca screen reader - In linux systems its often Super + alt + S

### Expected behavior

There is many elemnts missing alternative text and some is maby net needed that the screen reader reads

### Platform

Linux

### Browser

Firefox

### Jellyfin server version

10.10.0-unstable

**Comment by ferferga** [comment]
Thank you very much for opening an issue for this, this is something I'm aware of and definitely needs addressing (though it will take quite some time to make big steps in accessibility)
